Dockerfile.builder

```diff
@@ -17,7 +17,7 @@ RUN go install github.com/jstemmer/go-junit-report@latest
 
 ### Install NodeJS and yarn
 ENV NODE_VERSION="v14.16.0"
-ENV YARN_VERSION="v1.22.10"
+ENV YARN_VERSION="v1.22.19"
```

Comment:
Hey @rhamilto it's actually using the yarn version checked into the repo here:

https://github.com/openshift/console/tree/master/frontend/.yarn/releases

See the README here for steps to update:

https://github.com/openshift/console#frontend

Reply:
Thanks, @spadgett.  Updated.  🤞